apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -87,10 +87,13 @@ public function get_nux_status() {
 
 		$show_welcome_guide = $this->show_wpcom_welcome_guide( $nux_status );
 
+		$should_open_patterns_panel = (bool) get_option( 'is_blank_canvas' );
+
 		return rest_ensure_response(
 			array(
-				'show_welcome_guide' => $show_welcome_guide,
-				'variant'            => $variant,
+				'show_welcome_guide'         => $show_welcome_guide,
+				'variant'                    => $variant,
+				'should_open_patterns_panel' => $should_open_patterns_panel,
```

Comment:
When I made `variant` a string rather than just having a boolean like `show_tour: true` I imagined situations were there might be multiple variants like `variant` could be "modal" or "tour" or "tour v2" or "podcast tour".
I wonder whether this is like a variant too? Like `variant === "blank-canvas-tour"`? Maybe not, I haven't thought about it very much, just raising it as an option.

Reply:
Fixed in dc61f62